crates/datafusion_ext/src/planner/relation/mod.rs

```diff
@@ -250,10 +250,6 @@ fn infer_func_for_file(path: &str) -> Result<OwnedTableReference> {
             schema: "public".into(),
             table: "read_parquet".into(),
         },
-        "xlsx" => OwnedTableReference::Partial {
-            schema: "public".into(),
-            table: "read_excel".into(),
-        },
```

Comment:
Redundant case, handled again below: Just a fix for compiler warning.

Reply:
this was merged to main here: https://github.com/GlareDB/glaredb/commit/770f4e4a0122c562332525db38e03122556d8b14

Reply:
@vrongmeal we need to undo this diff now that you've merged main, as it will be deleting both arms

Reply:
Yup! I'll delete this.